js/script.js

```diff
@@ -97,9 +97,31 @@ console.log(getRandomQuote());
  * `printQuote` function
 ***/
 function printQuote () {
-  let message = "";
-  let showQuote = getRandomQuote(quotes);
-  //HI KEVIN, THIS IS WHERE I AM STUCK, SHOULD I DO A ELSE IF CLAUSE?
+  let display = getRandomQuote(quotes);
+  let message = `
+    <p class = "quote"> ${display.quote}</p>
+    <p class = "source"> ${display.source}</p>
+    <p class = "citation"> ${display.citation}</p>
+    <p class = "year"> ${display.year}</p>
+    <p class = "tag"> ${display.tag}</p> `
+  
+  if (display.quote) {
+    message += `<span class = "quote"> ${display.quote}</span>`
+  }    
+  if (display.source) {
+    message += `<span class = "source"> ${display.source}</span>`
+  }    
+  if (display.citation) {
+    message += `<span class = "citation"> ${display.citation}</span>`
+  }  
+  if (display.year) {
+    message += `<span class = "year">${display.year}</span>`
+  }
```

Comment:
so you should  do something like this... 
`const { quote, source, citation, year} = getRandomQuote(quotes);  
 const message = `
 <p class = "quote"> ${quote}</p>
     <p class = "source"> ${source}</p>
     <p class = "citation"> ${citation}</p>
     <p class = "year"> ${year}</p>
`;
`